core/app/[locale]/(default)/product/[slug]/page.tsx

```diff
@@ -28,16 +28,10 @@ const getOptionValueIds = ({ searchParams }: { searchParams: Awaited<Props['sear
     );
 };
 
-const getProduct = async (productPromise: ReturnType<typeof getProductData>) => {
```

Comment:
Split up monolith `getProduct` into individual `getAccordions`, `getTitle`, `getSubtitle`, `getRating`, `getPrice`, `getDescription`, and `getImages` streamable getters.